core-web/libs/edit-content/project.json

```diff
@@ -17,5 +17,5 @@
             "outputs": ["{options.outputFile}"]
         }
     },
-    "tags": []
+    "tags": ["type:feature", "scope:dotcms-ui", "portlet:edit-content"]
```

Comment:
I think we want to talk about this before it gets out of hand, this is an excellent feature that can get out of control very fast.

Reply:
I created this issue regarding this https://github.com/dotCMS/core/issues/28752